### ISSUE TITLE
I want to trade Synthetix and other DeFi products using CCXT

### ISSUE THREAD

**Issue by bitcoinmeetups** [ccxt/ccxt#6309]
I want to trade Synthetix and other DeFi products using CCXT

Hi, 

Feature request. I want to be able to trade Synthetix and do all the stuff that I use the Trust app for now using CCXT. Is there any way this is technically possible to implement?

I mean a lot of these new DeFi products are kind of fun to play around with and could become profitable but I don't want to use an Android app or a GUI, I want to use CLI like with CCXT.

Feedback please.

BM

**Comment by frosty00** [comment]
Yes it is possible, however we would need to use an ethereum explorer. I'm in favour for using etherscan to connect to the ethereum blockchain to support DEFI. However we have other priorities atm, namely #56

**Comment by bitcoinmeetups** [comment]
Please prioritize this if possible. DeFi is growing a lot. But all DeFi users are stuck using apps like Trust Wallet and a GUI instead of a CLI. I mean it's not only about being able to use Synthetix from the CLI, it's all of these Dapps like Synthetix, MakerDAO, LocalCryptos.com, Compound, Nuo, Kyber and more. You at team CCXT could provide the gateway to all of that.

**Comment by FIREFOXCYBER** [comment]
Is there any problem ahead of this feature?

**Comment by 50Shades0fShay** [comment]
has there been any progress on or around this?
